setup_ubuntu18.04.sh

```diff
@@ -4,11 +4,12 @@ SCRIPT_DIR=$(cd $(dirname $0); pwd)
 
 # Prevent root execution
 if [ $(id -u) -eq 0 ]; then
-  echo "Don't run as root" >&2
+  echo "Do not run setup as root!" >&2
   exit 1
 fi
 
-read -p ">  Do you run setup? This will take a while. [y/N] " answer
+echo "Setting up the pre-build environment for AutowareArchitectureProposal.IV can take up to 45 minutes."
```

Comment:
@LalithVipulananthan Please let me ask one question.
What did you mean by `pre-build environment`? What's the difference between just `build environment`?

Reply:
@kenji-miyake It made sense to me at the time, but looking at it again I think you're right. It should just say "build environment" instead of "pre-build environment"

Reply:
Thank you! Okay then, I'll fix it!